pkg/storage/system_intake_document.go

```diff
@@ -161,6 +163,7 @@ func (s *Store) DeleteSystemIntakeDocument(ctx context.Context, id uuid.UUID) (*
 		)
 		return nil, err
 	}
+	defer stmt.Close()
```

Comment:
Good catch! Generic note not just for this line, in mint we've created a helper function that wraps some of this boilerplate. `pkg/sqlutils/exec_procedure.go` `GetProcedure`

Not in the scope of this PR, but could be helpful to use for the future.!

Reply:
Oh! That looks really nice, we should definitely implement something like that in `easi` 😄